nailgun/entities.py

```diff
@@ -6814,8 +6814,12 @@ def __init__(self, server_config=None, **kwargs):
                 'remote_execution_proxy':
                     entity_fields.OneToManyField(SmartProxy),
                 'subnet_parameters_attributes': entity_fields.ListField(),
+                'template': entity_fields.OneToOneField(SmartProxy),
                 'tftp': entity_fields.OneToOneField(SmartProxy),
             })
+        if kwargs.get('_from'):
```

Comment:
we use opposite form for reserved words - `'from_'`. E.g. `DiscoveryRule.search_`

Reply:
Please test this fix with all CRUD operations, previously we had to implement custom read/update/create with such fix (like in DiscoveryRule), so i'm not so sure it will work properly for all cases.

Reply:
sure. looks like we might still have trouble using this with `.update`

Reply:
done, pls, resolve this if happy with the solution